.gitignore

```diff
@@ -5,3 +5,4 @@ count.out
 test
 profile.out
 tmp.out
+.idea
```

Comment:
please move it, thanks!

Reply:
It's goland configuration directory.

Reply:
thanks, we don't need it, we only persist this project related files.

Reply:
OK